src/samtools/flagstat/script.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+## VIASH START
+## VIASH END
+
+set -e
+
+output_file="$par_output"
+
+samtools flagstat \
+    "$par_bam" \
+    > "$output_file"
```

Comment:
Is there a reason for the `output_file` environment variable?

```suggestion
samtools flagstat "$par_bam" > "$par_output"
```

Reply:
No actually no reason, I forgot to remove it, thanks!